openshift-install-powervs

```diff
@@ -831,70 +832,82 @@ function variables_nodes {
     no_of_bastion=1
   fi
 
-  question "Do you want to use the default configuration for all the cluster nodes?" "yes no"
-  if [ "${value}" == "yes" ]; then
-    {
-      echo "bastion = {memory = \"16\", processors = \"1\", \"count\" = $no_of_bastion}"
-      echo "bootstrap = {memory = \"32\", processors = \"0.5\", \"count\" = 1}"
-      echo "master = {memory = \"32\", processors = \"0.5\", \"count\" = 3}"
-      echo "worker = {memory = \"32\", processors = \"0.5\", \"count\" = 2}"
-    } >> "$VAR_TEMPLATE"
-    return 0
-  fi
+  if [ -s $FLAVOR_TYPE ]; then
```

Comment:
Why such a big change with just spaces?
You can do...
```
if [[ ! .... ]]; then
    cat automation/compute-vars/$FLAVOR_TYPE.tfvars >> "$VAR_TEMPLATE"
    return 0
 fi
 ```

Reply:
Updated the file.